docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -309,6 +309,16 @@ The SiteAccess-aware `pagelayout` setting is deprecated in favor of `page_layout
 
 View parameter `pagelayout` set by `pagelayout` setting is deprecated in favor of  `page_layout`.
 
+#### Template paths
+
+Following the [upgrade to Symfony 4](#symfony-4), [the templating component integration is now deprecated.](https://symfony.com/blog/new-in-symfony-4-3-deprecated-the-templating-component-integration)
```

Comment:
**to doc-review:**
Optionally, I can add this link to the subsection on Symfony 4 (line 7). :)